modules/fbx/fbx_document.cpp

```diff
@@ -346,12 +346,11 @@ Error FBXDocument::_parse_nodes(Ref<FBXState> p_state) {
 		node.instantiate();
 
 		node->height = int(fbx_node->node_depth);
-
-		if (fbx_node->name.length > 0) {
+		if (fbx_node->is_root) {
```

Comment:
I don't think it's ok to always name the root node "RootNode". Previously we used what was the name in the fbx.

Reply:
sadly the fbx->rootname is almost always empty..
is it not implemented?

Reply:
> I don't think it's ok to always name the root node "RootNode". Previously we used what was the name in the fbx.

btw fbx_node->is_root slmost never returns true hmmmm...